app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -293,10 +314,320 @@ class QuestionPlayerActivityTest {
     }
   }
 
+  @RunOn(TestPlatform.ROBOLECTRIC)
```

Comment:
Why cannot we run the test on espresso too?

Reply:
I investigated on this a bit more and it seems like this was just an issue while testing on a real device.

Reply:
SOunds good, please also add espresso screenshot.